src/blist.rs

```diff
@@ -732,15 +732,15 @@ mod bench{
 
     #[bench]
     fn bench_collect_into(b: &mut test::Bencher) {
-        let v = &[0is; 64];
+        let v = &[0i32; 64];
         b.iter(|| {
-            let _: BList<isize> = v.iter().map(|x| *x).collect();
+            let _: BList<i32> = v.iter().map(|x| *x).collect();
         })
     }
 
     #[bench]
     fn bench_push_front(b: &mut test::Bencher) {
-        let mut m: BList<isize> = BList::new();
+        let mut m: BList<i32> = BList::new();
```

Comment:
collect/src/blist.rs:744:11: 746:11 error: unable to infer enough type information about `closure[collect/src/blist.rs:744:16: 746:10]`; type annotations required